### PR TITLE
structures added

### PR DESCRIPTION
**Pull request by HembramBeta777**
classes were removed and structures added.

### CODE REVIEW

File: bmp.h [hunk 1]
```diff
@@ -1,41 +1,41 @@
-#include <cstdint>
+
+#include <cstdint.h>
```
Comment (Himanshu40) [inline bmp.h:2]:
```suggestion
#include <cstdint>
```
Why you added .h?

File: bmp.h [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
 
 typedef uint8_t BYTE;
 typedef uint32_t DWORD;
 typedef int32_t LONG;
 typedef uint16_t WORD;
```
Comment (Himanshu40) [inline bmp.h:7]:
```suggestion
typedef uint8_t   BYTE;
typedef uint32_t DWORD;
typedef int32_t   LONG;
typedef uint16_t WORD;
```
Follow a coding style standard.

File: bmp.h [hunk 3]
```diff
@@ -1,41 +1,41 @@
...
+} __attribute__((__packed__))
+BITMAPINFOHEADER;
+
+typedef struct
+{
+    BYTE rgbtBlue;
+    BYTE rgbtGreen;
+    BYTE rgbtRed;
+} __attribute__((__packed__))
+RGBTRIPLE;
```
Comment (Himanshu40) [inline bmp.h:41]:
```suggestion
RGBTRIPLE;

```
Add a new line.

File: bmp.h [hunk 4]
```diff
@@ -1,41 +1,41 @@
...
+    DWORD biSizeImage;
+    LONG biXPelsPerMeter;
+    LONG biYPelsPerMeter;
+    DWORD biClrUsed;
+    DWORD biClrImportant;
+} __attribute__((__packed__))
+BITMAPINFOHEADER;
+
+typedef struct
+{
```
Comment (Himanshu40) [inline bmp.h:36]:
```suggestion
typedef struct {
```

File: bmp.h [hunk 5]
```diff
@@ -1,41 +1,41 @@
...
+    WORD bfType;
+    DWORD bfSize;
+    WORD bfReserved1;
+    WORD bfReserved2;
+    DWORD bfOffBits;
+} __attribute__((__packed__))
+BITMAPFILEHEADER;
+
+typedef struct
+{
```
Comment (Himanshu40) [inline bmp.h:20]:
```suggestion
typedef struct {
```

File: bmp.h [hunk 6]
```diff
@@ -1,41 +1,41 @@
...
-
-
-class RGBTRIPLE {
-    public:
-        BYTE rgbtBlue;
-        BYTE rgbtGreen;
-        BYTE rgbtRed;
-} __attribute__((__packed__));
+typedef struct
+{
```
Comment (Himanshu40) [inline bmp.h:10]:
```suggestion
typedef struct {
```
Follow this bracket style.

**Review by Himanshu40 — CHANGES_REQUESTED** [review]

**Comment by chandrakant100** [comment]
Do follow Proper Indentation and give credits to Microsoft in the comments because we are using their Structure.

File: bmp.h [hunk 7]
```diff
@@ -32,8 +31,7 @@ typedef struct
 } __attribute__((__packed__))
 BITMAPINFOHEADER;
 
-typedef struct
-{
+typedef struct{
```
Comment (Himanshu40) [inline bmp.h]:
```suggestion
typedef struct {
```

File: bmp.h [hunk 8]
```diff
@@ -16,8 +16,7 @@ typedef struct
 } __attribute__((__packed__))
 BITMAPFILEHEADER;
 
-typedef struct
-{
+typedef struct{
```
Comment (Himanshu40) [inline bmp.h]:
```suggestion
typedef struct {
```

File: bmp.h [hunk 9]
```diff
@@ -1,13 +1,13 @@
...
 typedef uint8_t BYTE;
 typedef uint32_t DWORD;
-typedef int32_t LONG;
 typedef uint16_t WORD;
+typedef int32_t LONG;
+
 
-typedef struct
-{
+typedef struct{
```
Comment (Himanshu40) [inline bmp.h]:
```suggestion
typedef struct {
```
Please add space after struct.

**Review by Himanshu40 — CHANGES_REQUESTED** [review]

**Comment by chandrakant100** [comment]
Evrything is Fine now Ready to merge!

File: bmp.h [hunk 10]
```diff
@@ -1,41 +1,72 @@
...
+ * red, green, and blue.
+ *
+ * Adapted from http://msdn.microsoft.com/en-us/library/aa922590.aspx.
+ */
+typedef struct {
+    BYTE  rgbtBlue;
+    BYTE  rgbtGreen;
+    BYTE  rgbtRed;
+} __attribute__((__packed__))
+RGBTRIPLE;
```
Comment (Himanshu40) [inline bmp.h:72]:
```suggestion
RGBTRIPLE;

```
Add new line please.

**Review by Himanshu40 — CHANGES_REQUESTED** [review]
Everything is fine. Just add a new line.

**Comment by Himanshu40** [comment]
@chandrakant100 Merge this PR.

**Pull request merged**